validator/src/main.rs

```diff
@@ -1443,9 +1443,9 @@ pub fn main() {
                 .help("Enable an accounts index, indexed by the selected account field"),
         )
         .arg(
-            Arg::with_name("accounts_db_caching_enabled")
```

Comment:
Can we leave this in as a hidden NOP arg?

Reply:
Yes please, `.hidden(true)`.  Otherwise the roll out will be painful to our nodes